src/flac/main.c

```diff
@@ -1278,98 +1278,109 @@ void show_help(void)
+	printf("  -l, --max-lpc-order=#              Max LPC order; 0 => only fixed predictors\n");
 	printf("  -b, --blocksize=#                  Specify blocksize in samples\n");
 	printf("  -m, --mid-side                     Try mid-side coding for each frame\n");
-	printf("  -M, --adaptive-mid-side            Adaptive mid-side coding for all frames\n");
+	printf("  -M, --adaptive-mid-side            Adaptive choice of mid-side coding\n");
+	printf("  -r, --rice-partition-order=[#,]#   Set [min,]max residual partition order\n");
+	printf("  -A, --apodization=\"function\"       Window audio data with given function(s)\n");
 	printf("  -e, --exhaustive-model-search      Do exhaustive model search (expensive!)\n");
-	printf("  -A, --apodization=\"function\"       Window audio data with given the function\n");
-	printf("  -l, --max-lpc-order=#              Max LPC order; 0 => only fixed predictors\n");
+	printf("  -q, --qlp-coeff-precision=#        Specify quantization precision in bits\n");
+	printf("                                     (default: let encoder decide)\n");	
 	printf("  -p, --qlp-coeff-precision-search   Exhaustively search LP coeff quantization\n");
-	printf("  -q, --qlp-coeff-precision=#        Specify precision in bits\n");
-	printf("  -r, --rice-partition-order=[#,]#   Set [min,]max residual partition order\n");
+	printf("      --lax                          Allow encoder to generate non-Subset files\n");
 	printf("      --limit-min-bitrate            Limit minimum bitrate (for streaming)\n");
 	printf("  -j, --threads=#                    Set number of encoding threads\n");
-	printf("format options:\n");
-	printf("      --force-raw-format             Treat input or output as raw samples\n");
+	printf("      --ignore-chunk-sizes           Ignore data chunk sizes in WAVE/AIFF files\n");
+	printf("      --replay-gain                  Calculate ReplayGain & store in FLAC tags\n");
+	printf("      --cuesheet=FILENAME            Import cuesheet & store in CUESHEET block\n");
+	printf("      --picture=SPECIFICATION        Import picture & store in PICTURE block\n");
+	printf("  -T, --tag=FIELD=VALUE              Add a FLAC tag; may appear multiple times\n");
+	printf("      --tag-from-file=FIELD=FILENAME     Like --tag but gets value from file\n");
+	printf("  -S, --seekpoint={#|X|#x|#s}        Add seek point(s)\n");
+	printf("  -P, --padding=#                    Write a PADDING block of length # bytes\n");
+	printf("\n");
+	printf("Format options (encoding defaults to FLAC not OGG; decoding defaults to WAVE, \n");
+	printf("             chunks found by --keep-foreign-metadata-if-present will override):\n");
+#if FLAC__HAS_OGG
+	printf("      --ogg                          Use Ogg transport layer, output .oga\n");
+	printf("      --serial-number                Ogg serial number to assign (encoding)\n);
+	printf("                                     or to select for decoding\n);
```

Comment:
```suggestion
	printf("                                     or to select for decoding\n");
```
The line above it is missing one as well

Reply:
You clicked resolve without resolving it first. No worries, I did it just now, and this is still fun. 😄